1-org/modules/centralized-logging/versions.tf

```diff
@@ -16,4 +16,11 @@
 
 terraform {
   required_version = ">= 1.3"
+
+  required_providers {
```

Comment:
Do we want to add this provider?  Doesn't appear be a Google or HashiCorp supported.

Reply:
Usage of this was a recommendation from @bharathkkb to call the Google Cloud Logging  API directly instead of using the gcloud module.

This module is presented in an official post in the Hashicorp site blog as the suggestion to make managed and unmanaged API calls for resources not supported by a provider.

https://www.hashicorp.com/blog/writing-terraform-for-unsupported-resources

Reply:
it is also used in the cloud run module 
https://github.com/GoogleCloudPlatform/terraform-google-cloud-run/blob/main/modules/job-exec/main.tf#L99

Reply:
Thanks @daniel-cit - Once @gtsorbo or @eeaton review, I'll get this merged.

Reply:
I've reviewed, the functional changes are LGTM